colour/models/rgb/conversion_functions/bt_1886.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+ITU-R BT.1886 EOTF / EOCF
+=========================
+
+Defines *Recommendation ITU-R BT.1886* EOTF / EOCF:
+
+-   :func:`BT_1886_EOCF`
+
+See Also
+--------
+`ITU-R BT.1886 EOTF / EOCF IPython Notebook
+<http://nbviewer.ipython.org/github/colour-science/colour-ipython/\
+blob/master/notebooks/models/bt_1886.ipynb>`_
+
+References
+----------
+.. [1]  International Telecommunication Union. (2011). Recommendation ITU-R
+        BT.1886 - Reference electro-optical transfer function for flat panel
+        displays used in HDTV studio production BT Series Broadcasting service.
+"""
+
+from __future__ import division, unicode_literals
+
+import numpy as np
+
+__author__ = 'Colour Developers'
+__copyright__ = 'Copyright (C) 2013 - 2015 - Colour Developers'
+__license__ = 'New BSD License - http://opensource.org/licenses/BSD-3-Clause'
+__maintainer__ = 'Colour Developers'
+__email__ = 'colour-science@googlegroups.com'
+__status__ = 'Production'
+
+__all__ = ['BT_1886_EOCF']
+
+
+def BT_1886_EOCF(V, L_W=940, L_B=64):
```

Comment:
Yes, this is problematic, I'll address that as a whole when doing the naming convention consistency work.